examples/user_guide/Plotting_with_Matplotlib.ipynb

```diff
@@ -0,0 +1,672 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we discovered in the [Introduction](Introduction.ipynb), HoloViews allows plotting a variety of data types. Here we will use the sample data module and load the pandas and dask hvPlot API:"
```

Comment:
I would add a sentence along the lines of 'This notebook demonstrates the use of the Matplotlib plotting backend, the equivalent notebook demonstrating the Plotly backend may be found HERE (link)'